### PR TITLE
Fixed controller hiding when seeking in native player

### PR DESCRIPTION
**Pull request by yashx**
### Before
old

### Now
new

### CODE REVIEW

File: app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt [hunk 1]
```diff
@@ -51,6 +51,12 @@ class PlayerActivity : AppCompatActivity() {
      */
     private val hideUnlockButtonAction = Runnable { unlockScreenButton.isVisible = false }
 
+    /**
+     * Runnable that hides [playerView] controller
+     */
+    private val hidePlayerViewControllerAction = Runnable { playerView.hideController() }
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt:57]:
```suggestion
    private val hidePlayerViewControllerAction = Runnable {
        playerView.hideController()
    }
```

File: app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt [hunk 2]
```diff
@@ -51,6 +51,12 @@ class PlayerActivity : AppCompatActivity() {
...
+
+
```
Comment (nielsvanvelzen) [inline app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt:59]:
```suggestion

```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt (hunk 1 above)
Comment (Maxr1998) [reply app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt:57]: I'd say that's fine, doesn't really need extra wrapping imo. The line above also doesn't do that ^^

File: app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt (hunk 2 above)
Comment (nielsvanvelzen) [reply app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt]: I should've been more clear on this one; there were two empty lines and it should've been just one. You now removed both so you need to add one back.

Comment (yashx) [reply app/src/main/java/org/jellyfin/mobile/player/PlayerActivity.kt]: Oh. I didn't see what this was for. I just committed all the suggestions on Github and squashed the commit.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by Maxr1998 — APPROVED** [review]

**Pull request merged**